### PR TITLE
Update language

### PR DESCRIPTION
**Pull request by Krokit**
Updated Bulgarian language.

### CODE REVIEW

File: src/main/resources/messages/messages_bg.yml [hunk 1]
```diff
@@ -1,105 +1,104 @@
...
+add_email: '&3Моля добавете имейл адрес към своят акаунт: /email add имейл имейл'
+recovery_email: '&3Забравена парола? Използвайте: /email recovery имейл'
+email_cooldown_error: '&cВече е бил изпратен имейл адрес. Трябва а изчакаш %time преди да пратиш нов.'
 
 # Captcha
-usage_captcha: '&cYou need to type a captcha, please type: /captcha <theCaptcha>'
-wrong_captcha: '&cГрешен код, използвай : /captcha THE_CAPTCHA'
-valid_captcha: '&cТвоя код е валиден!'
+usage_captcha: '&3Моля въведе цифрите/буквите от капчата: /captcha <theCaptcha>'
+wrong_captcha: '&cКода е грешен, използвайте: "/captcha THE_CAPTCHA" into the chat!'
```
Comment (sgdc3) [inline src/main/resources/messages/messages_bg.yml]: Missing translation

**Comment by ljacqu** [comment]
hi @Krokit , thanks for this PR. Have you seen the comment by @sgdc3? We'll keep this open so you can finish translating `wrong_captcha`.

**Comment by Krokit** [comment]
Now leanguage is ready at 100%.

File: src/main/resources/messages/messages_bg.yml [hunk 2]
```diff
@@ -1,105 +1,104 @@
...
-# TODO accounts_owned_self: 'You own %count accounts:'
-# TODO accounts_owned_other: 'The player %name has %count accounts:'
-# TODO two_factor_create: '&2Your secret code is %code. You can scan it from here %url'
-# TODO recovery_code_sent: 'A recovery code to reset your password has been sent to your email.'
-# TODO recovery_code_incorrect: 'The recovery code is not correct! Use "/email recovery [email]" to generate a new one'
-vb_nonActiv: '&fТвоята регистрация не е активирана, моля провери своя Имейл!'
+unregistered: '&cРегистрацията е премахната успешн.!'
+accounts_owned_self: 'Претежаваш %count акаунт/а:'
+accounts_owned_other: 'Потребителят %name има %count акаунт/а:'
+TODO two_factor_create: '&2Кода е %code. Можеш да го провериш оттука: %url'
```
Comment (ljacqu) [inline src/main/resources/messages/messages_bg.yml]: Oops :)

Comment (sgdc3) [inline src/main/resources/messages/messages_bg.yml]: '.!' should be '!'

**Comment by sgdc3** [comment]
Thank you @Krokit ;)

**Pull request merged**